manifests-dev/deployment-toktik-auth-api.yaml

```diff
@@ -1,50 +1,50 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  annotations:
-    sidecar.jaegertracing.io/inject: 'true'
-  labels:
-    app: toktik-auth-api
-  name: toktik-auth-api
-  namespace: toktik-service-bundle
-spec:
-  selector:
-    matchLabels:
-      name: toktik-auth-api
-  template:
-    metadata:
-      labels:
+    annotations:
```

Comment:
@shaokeyibb @ExerciseBook @lss233 @Selflocking Please check if I destroyed the yaml ident.